lib/admin_page/project/project.dart

```diff
@@ -0,0 +1,181 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:web/admin_page/project/prject_list_widget.dart';
+import 'package:web/model/project_model.dart';
+
+import '../../values/app_assets.dart';
+
+class ProjectPage extends StatefulWidget {
+  const ProjectPage({Key? key}) : super(key: key);
+
+  @override
+  State<ProjectPage> createState() => _ProjectPageState();
+}
+
+class _ProjectPageState extends State<ProjectPage> {
+  var projectData = <ProjectDetail>[];
+  var searchProjectList = <ProjectDetail>[];
+  var type = "";
+  var occupation = "";
+  var searchController = TextEditingController();
+
+  @override
+  void initState() {
+    super.initState();
+    readJsonUserList();
+  }
+
+  String dropdownValue = 'PROJECT STATUS';
+  var items = ['PROJECT NAME', 'PROJECT STATUS', "NAME"];
+
+  readJsonUserList() async {
```

Comment:
Move the api call or fetching the data outside of view or page UI. Even if you are just mocking the data then also they should be part of your controllers/presenters depending on the State Managment Techniques you are using. Your UI is only responsible for rendering the data you provide to it.